src/components/Course.jsx

```diff
@@ -0,0 +1,59 @@
+// ============================================================================================
+// IMPORTS
+// ============================================================================================
+
+import PropTypes from "prop-types";
+import Module from "./Module.jsx";
+import Assessment from "./Assessment.jsx";
+
+import "./Course.css";
+
+// ============================================================================================
+// COMPONENT DEFINITION
+// ============================================================================================
+
+/*********************************************************************************************/
+
+function Course({ courseData }) {
+  return (
+    <div id="Course">
+      <h1>{courseData.title}</h1>
+      <p>
+        <aside style={{ float: "inline-end" }}>
```

Comment:
Avoid inline styles where possible to maintain a clear separation of concerns. Use CSS classes instead.

Reply:
Agreed.